mappings/net/minecraft/world/biome/source/VoronoiBiomeAccessType.mapping

```diff
@@ -1,13 +1,45 @@
 CLASS net/minecraft/class_4546 net/minecraft/world/biome/source/VoronoiBiomeAccessType
+	COMMENT The voronoi biome access type distorts the biome volume's voxel points
+	COMMENT and finds the closest distorted biome voxel point from a block position
+	COMMENT as if in a euclidean voronoi diagram and returns the biome stored in that
+	COMMENT biome voxel point.
+	COMMENT
+	COMMENT <p>The resulting biome distribution of the voronoi biome access type is
+	COMMENT like a raster of 3d voronoi distribution, hence the name.
+	COMMENT
+	COMMENT @see HorizontalVoronoiBiomeAccessType
 	METHOD method_22397 sqr (D)D
+		COMMENT Returns the square of {@code d}.
 		ARG 0 d
-	METHOD method_22398 distribute (J)D
+	METHOD method_22398 calculateShift (J)D
+		COMMENT Calculates the shift, in the biome voxel's axis unit, of the weight
+		COMMENT center of a biome position from the regular grid position in an axis.
+		COMMENT
+		COMMENT <p>It takes the 25th to 34th lowest bits of the seed, divides it by
+		COMMENT 1024, minuses it by 0.5, and multiply it by 0.9 to get a shift between
```

Comment:
```suggestion
		COMMENT 1024, subtracts 0.5 from it, and multiply it by 0.9 to get a shift between
```

Reply:
why?

Reply:
`minuses` isn't an operation, it's subtraction.

Reply:
Done